editor.html

```diff
@@ -183,34 +183,45 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
-                        <div class="helpsupport"><a id="help-link" class="action" title="Open help for this editor in a new tab" href="help.html" target="_blank" >Help</a></div>
-                        <div class="helpsupport"><a id="support-link" class="action" title="Get support for your micro:bit in a new tab" href="https://support.microbit.org/support/home" target="_blank">Support</a></div>
-                        <div class="helpsupport hidden" id="known-issues"><a id="issues-link" class="action" title="View open issues for the Python Editor in GitHub" href="https://github.com/bbcmicrobit/PythonEditor/issues" target="_blank">Issue Tracker</a></div>
+                <div class="buttons_menu_container hidden" id="helpsupport_container">
+                    <div class="buttons_menu_lhs">
+                        <div class="buttons_menu_item"><a id="docs-link" class="action" title="View the documentation for MicroPython" href="https://microbit-micropython.readthedocs.io/" target="_blank">Documentation</a></div>
+                        <div class="buttons_menu_item"><a id="help-link" class="action" title="Open help for this editor in a new tab" href="help.html" target="_blank" >Help</a></div>
+                        <div class="buttons_menu_item"><a id="support-link" class="action" title="Get support for your micro:bit in a new tab" href="https://support.microbit.org/support/home" target="_blank">Support</a></div>
+                        <div class="buttons_menu_item hidden" id="known-issues"><a id="issues-link" class="action" title="View open issues for the Python Editor in GitHub" href="https://github.com/bbcmicrobit/PythonEditor/issues" target="_blank">Issue Tracker</a></div>
                     </div>
-                    <div class="helpsupport_rhs">
-                        <div class="helpsupport">Editor Version: <script>document.write(EDITOR_VERSION);</script></div>
-                        <div class="helpsupport">MicroPython Version: <script>document.write(UPY_VERSION);</script></div>
+                    <div class="buttons_menu_rhs">
+                        <div class="buttons_menu_item">Editor Version: <script>document.write(EDITOR_VERSION);</script></div>
+                        <div class="buttons_menu_item">MicroPython Version: <script>document.write(UPY_VERSION);</script></div>
                     </div>
                 </div>
-                <a href="#" class="roundbutton action hidden"
-                    id="command-share" tabindex="7"
-                    title="Create a link to share your script">
-                    <div class="roundsymbol">
-                        <i class="fa fa-share-alt"></i>
-                    </div>
-                    <div class="roundlabel">Share</div>
-                </a>
```

Comment:
It's not sprightly necessary for this PR, but I've moved the share button to be placed __before__ the options and help buttons.

Reply:
Is there a particular reason?

Reply:
If it's enabled the share button ends up after the help button (and before the zoom buttons). Help should always be the left-most button as that is a common UI pattern.